src/send_kcidb.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Maryam Yusuf
+# Author: Maryam Yusuf <maryam.m.yusuf1802@gmail.com>
+import datetime
+import os
+import sys
+
+import kernelci
+import kernelci.data
+from kernelci.config import load
+from kernelci.cli import Args, Command, parse_opts
+from kcidb import Client
+import kcidb
+
+
+class cmd_run(Command):
+    help = "Listen for events and send them to KCDIB"
+    args = [Args.db_config]
+    project_id = os.getenv('KCIDB_PROJECT_ID')
+    topic_name = os.getenv('KCIDB_TOPIC_NAME')
+    google_credential = os.getenv('GOOGLE_APPLICATION_CREDENTIALS')
+    client = None
+
+    if topic_name and project_id:
+        client = Client(project_id=project_id,
+                        topic_name=topic_name)
```

Comment:
I think this code should be moved to `__call__()` as the class initialisation is global to all object instances whereas this is per-instance code.  I know there's only one object instance in this case, but it's still an issue from a design point of view.  If we want to add unit tests or run multiple clients in parallel it will become an issue.